feature/mpls/sr/otg_tests/isis_node_sid_forward/README.md

```diff
@@ -0,0 +1,67 @@
+# SR-1.1: Transit forwarding to Node-SID via ISIS
+
+## Summary
+
+MPLS-SR transit forwarding to Node-SID distributed over ISIS
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Configuration
+
+Topology: ATE1—DUT1–ATE2
+                               
+*   Configure Segment Routing Global Block (start: 400000 range: 65001)
+*   Enable Segment Routing for the ISIS
+*   Enable MPLS forwarding (if required)
+
+*  Prefix (1) with node-SID is advertised by the direct ISIS neighbor
+*  Prefix (2) with node-SID is advertised by simulated indirect ISIS speaker
+
+### Test 
+
+Verify that:
+
+*  DUT receives prefixes with SIDs and has a correct emulated ISIS topology (database).
```

Comment:
I think verifying DUT aggregate-sid-counters/aggregate-sid-counter/state/(in|out)-pkts is a good idea.  

But regarding DUT ISIS lsdb state, I think we should instead rely on traffic receipt at the OTG being correct. This would be much more simple and doesn't require trusting the DUT lsdb state.  (generally want to verify things via OTG, and minimize dependency on 'test-only' DUT specific paths). If you are ok with this, please remove the 'verify correct emulated ISIS topology' part of this verification.


(unless we expect to actually use ISIS lsdb leafs in production.  If yes, then we need to specify which lsdb leafs and how to query them (just for presence or if we require some specific value verification)).

Reply:
Makes sense. Removed.